lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -298,14 +292,34 @@ def add_framework_file_reference_to_native_target(native_target, pod_target, dep
           end
         end
 
-        # Sets the APPLICATION_EXTENSION_API_ONLY build setting to YES for all
-        # configurations of the given native target.
-        #
         def configure_app_extension_api_only_to_native_target(native_target)
           native_target.build_configurations.each do |config|
             config.build_settings['APPLICATION_EXTENSION_API_ONLY'] = 'YES'
           end
         end
+
+        def configure_schemes_for_pod_target(project, pod_target)
+          pod_target.specs.each do |spec|
```

Comment:
I just thought of something............ what happens on subspec scheme configuration?! Today all schemes are merged to the top (root) I believe

Reply:
still need to address this.

Reply:
- [ ] fix this

Reply:
Fixed here https://github.com/CocoaPods/Core/pull/479